packages/mobile/src/config/linking/config.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  RootScreenName,
+  MainScreenName,
+  WelcomeRootScreenName,
+  DrawerScreenName,
+  DeepLinkingPaths,
+} from '~/common/enums/enums';
+
+const config = {
+  screens: {
+    [RootScreenName.MAIN_WITH_MENU]: {
+      screens: {
+        [DrawerScreenName.MAIN]: {
+          screens: {
+            [MainScreenName.HOME]: DeepLinkingPaths.HOME,
+            [MainScreenName.WELCOME_ROOT]: {
+              screens: {
+                [WelcomeRootScreenName.SIGN_IN]: DeepLinkingPaths.SIGN_IN,
+                [WelcomeRootScreenName.SIGN_UP]: DeepLinkingPaths.SIGN_UP,
+              },
+            },
+          },
+        },
+      },
+    },
+    [RootScreenName.ITEM_INFO]: {
+      path: DeepLinkingPaths.ITEM,
```

Comment:
Let's use strings explicitly without enum. It would be more obvious where you extract params